guest/rust/examples/basics/decals/src/server.rs

```diff
@@ -0,0 +1,57 @@
+use ambient_api::{
+    components::core::{
+        app::main_scene,
+        camera::aspect_ratio_from_window,
+        primitives::{quad, cube},
+        rendering::{color, transparency_group},
+        transform::{lookat_target, rotation, scale, translation},
+    },
+    concepts::{make_perspective_infinite_reverse_camera},
+    prelude::*,
+};
+
+use core::f32::consts::PI;
+
+// from discussion at
+// https://discord.com/channels/894505972289134632/1078283561540530216/1096581219925377195
+
+#[main]
+pub fn main() {
+    Entity::new()
+        .with_default(cube())
+        .with(translation(), Vec3::Z)
+        .with(color(), vec4(0.5, 0.5, 0.5, 1.))
+        .with_default(cast_shadows())
+        .spawn();
+
+    Entity::new()
+        .with_default(quad())
+        .with(scale(), Vec3::ONE * 10.)
+        .spawn();
+
+    Entity::new()
```

Comment:
I think since we're anyway adding the cube stuff in the system above, you might not need the `.with_default(cube())` here?

Reply:
removed in [b8ed8aa](https://github.com/AmbientRun/Ambient/pull/347/commits/b8ed8aa69d36ea3c6f01c24914faca4e91d5f91b)